src/Fields/field_reductions.jl

```diff
@@ -13,21 +13,35 @@ end
 
 Return a `Reduction` of `operand` with `reduce!`, along `dims`.
 
-elements for which `condition(i ,j ,k) == true` are masked with a `mask`
+Field elements for which `condition(i ,j ,k) == true` are masked with a `mask`.
 
 Example
```

Comment:
Should we mention somewhere here that the reduction must happen in-place? (Or is that not the case?)

Reply:
you mean that the reduction operator must be a function that works 'in place'?

Reply:
Yes!

Reply:
Done via  9713191